AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenInfo.h

```diff
@@ -7,15 +7,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface MSAuthTokenInfo : NSObject
 
-@property(nonatomic, nullable,  copy, readonly) NSString *authToken;
+@property(nonatomic, nullable, copy, readonly) NSString *authToken;
 
-@property(nonatomic, nullable, readonly) NSDate *startTime;
-
-@property(nonatomic, nullable, readonly) NSDate *endTime;
+@property(nonatomic, readonly) int timestamp;
```

Comment:
This class was planned for returning result and requires 2 time-fields. Please revert